### PR TITLE
Issues/k UI 1599 remove fields

### PR DESCRIPTION
**Pull request by amirhossein-haerian**
Följande rubriker ska inte längre visas på sidan Inför kursval:

Kontaktperson (KoppsAPI: infoContactName)

Påbyggnad (KoppsAPI: addOn)

Information till forskarstuderande om när kursen ges (KoppsAPI: applicationInfo)

Möjlighet till komplettering (KoppsAPI:possibilityToCompletion)

Möjlighet till plussning (KoppsAPI:possibilityToAddition)

Utrustning (KoppsAPI:requiredEquipment)

### CODE REVIEW

File: server/controllers/createSyllabusList.js
Comment (amirhossein-haerian) [inline server/controllers/createSyllabusList.js:1]: I assume that the `requiredEquipment` field should be removed from the syllabusList as well. Let me know if we should not remove it.

Comment (karlandindrakryggen) [reply server/controllers/createSyllabusList.js:1]: Good question! I'm not sure but will look into it.

Comment (karlandindrakryggen) [reply server/controllers/createSyllabusList.js:1]: As I said, I believe that you done this correctly, but I added a question to Susanne in the parent Jira issue just to be sure (KUI-1387).

**Review by karlandindrakryggen — APPROVED** [review]

File: server/controllers/createSyllabusList.js
Comment (amirhossein-haerian) [reply server/controllers/createSyllabusList.js:1]: Thanks for following this up. That would be great for us to be sure.

**Review by allazis — APPROVED** [review]
LGTM! ⭐

**Review by axelbjo — APPROVED** [review]
LGTM!

**Pull request merged**